src/content/getting-started/quickstart/kind/_index.md

```diff
@@ -23,12 +23,14 @@ To create kind clusters and deploy Submariner, run:
 ```bash
 git clone https://github.com/submariner-io/submariner-operator
 cd submariner-operator
-make deploy
+make deploy using=lighthouse
```

Comment:
We should likely make the summary of these steps reflect the updated steps, something like (can't suggest on the relevant line because it isn't edited, sorry):

> To create kind clusters and deploy Submariner with service discovery, run: